### DISCUSSION TITLE
Vehicles Megathread

### DISCUSSION THREAD

**Discussion by CubicRP** [CubicRP/Cubic#7] [Ideas]
Suggestions/issues regarding vehicles. 

(New vehicles, handling changes, vehicle bugs etc - new scripts can be discussed in new discussions)

**Comment by SnowGambit** [comment]
What kind of street bikes have been added so far?

**Reply by dwreyyes** [reply]
Right now nothing with vehicles is final. I think there are still additions being made and taken away. That being said, I am not sure there is a list anywhere as to what is in and what isn't.

**Comment by UGotCuteToes** [comment]
Could you guys add a ninja H2r for a motorcycle. Also can you add in a 370z with a lot of body work detail? such as a black hood, widebody kit, some fender, bumpers, spoiler and such please.

**Reply by LittleEgg0** [reply]
This is something we can look into, we already have a few high-end motorbikes.

We have a large range overall, but there will be several high-end bikes coming _release is TBD_

**Comment by Gibuso** [comment]
Vehicle sound for the dodge chargers (PD and regular) if possible :)
https://www.gta5-mods.com/vehicles/dodge-charger-6-4-6-2sc-v8-engine-sound-oiv-add-on-fivem-sound

**Reply by LittleEgg0** [reply]
We do plan to look into vehicle sounds once we have a good selection of highly optimized, highly detailed models.

**Comment by Kumotta** [comment]
> Any and all suggestions/issues regarding vehicles.

Need to add some sort of ability to tune your cars yourself with a mechanic or on your own to ecnhance the street racing community i personally would love this idea as well as Add as many JDM and German cars as possible
